lib/plugins/max-scroll-tracker.js

```diff
@@ -202,8 +202,9 @@ class MaxScrollTracker {
    * @return {number}
    */
   getPagePath() {
-    return parseUrl(
-        this.tracker.get('page') || this.tracker.get('location')).path;
+    const url = parseUrl(
```

Comment:
Yeah, I think this is just a Github error. It's not showing the correct base branch for some reason. When I merge it won't be in the history.